connmgr/tor.go

```diff
@@ -56,13 +56,6 @@ var (
 	}
 )
 
-// TorLookupIP uses Tor to resolve DNS via the passed SOCKS proxy.
-//
-// Deprecated: use TorLookupIPContext instead.
-func TorLookupIP(host, proxy string) ([]net.IP, error) {
-	return TorLookupIPContext(context.Background(), host, proxy)
-}
-
 // TorLookupIPContext uses Tor to resolve DNS via the passed SOCKS proxy.
 func TorLookupIPContext(ctx context.Context, host, proxy string) ([]net.IP, error) {
```

Comment:
perhaps rename this to simply TorLookupIP? otherwise looks good.

Reply:
That sounds good.  I'll open another PR to do the rename though so this one stays scoped to removing the deprecated funcs.

Reply:
Done in https://github.com/decred/dcrd/pull/2191.